k8s/k8s.go

```diff
@@ -3827,6 +3830,49 @@ func (k *k8sOps) DeleteClusterDomainsStatus(name string) error {
 	})
 }
 
+func (k *k8sOps) ValidateClusterDomainsStatus(name string, domainMap map[string]bool, timeout, retryInterval time.Duration) error {
```

Comment:
Don't you need a Validate for ClusterDomainUpdate also to make sure that it gets applied successfully?

Reply:
there is one already.